src/pages/iou/request/step/IOURequestStepParticipants.js

```diff
@@ -109,7 +118,14 @@ function IOURequestStepParticipants({
     const goToNextStep = useCallback(
         (selectedIouType) => {
             const isSplit = selectedIouType === CONST.IOU.TYPE.SPLIT;
-            const nextStepIOUType = !isSplit && iouType !== CONST.IOU.TYPE.REQUEST ? CONST.IOU.TYPE.REQUEST : iouType;
+            let nextStepIOUType;
```

Comment:
NAB: It might be a little better to default this to `CONST.IOU.TYPE.REQUEST` and then you can remove the ternary below.